sdk/healthinsights/azure-health-insights-cancerprofiling/README.md

```diff
@@ -0,0 +1,218 @@
+
+patient1.getData().add(patientDocument3);
+
+// Set configuration to include evidence for the cancer staging inferences
+OncoPhenotypeModelConfiguration configuration = new OncoPhenotypeModelConfiguration();
+configuration.setIncludeEvidence(true);
+
+// Construct the request with the patient and configration
+OncoPhenotypeData oncoPhenotypeData = new OncoPhenotypeData(Arrays.asList(patient1));
+oncoPhenotypeData.setConfiguration(configuration);
+
+PollerFlux<OncoPhenotypeResult, OncoPhenotypeResult> asyncPoller = asyncClient.beginInferCancerProfile(oncoPhenotypeData);
+```
+
+## Troubleshooting
+
+## Next steps
+<!--
+This code sample show common scenario operation with the Azure Health Insights Cancer Profiling library. More samples can be found under the [samples](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/healthinsights/azure-health-insights-cancerprofiling/src/samples/java/com/azure/health/insights/) directory.
+-->
+
+## Additional documentation
+For more extensive documentation on Azure Health Insights Cancer Profiling, see the [Cancer Profiling documentation][cancer_profiling_docs] on docs.microsoft.com.
+## Contributing
+
+This project welcomes contributions and suggestions. Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution. For details, visit [cla.microsoft.com][cla].
+
+When you submit a pull request, a CLA-bot will automatically determine whether you need to provide a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions provided by the bot. You will only need to do this once across all repos using our CLA.
+
+This project has adopted the [Microsoft Open Source Code of Conduct][code_of_conduct]. For more information see the [Code of Conduct FAQ][coc_faq] or contact [opencode@microsoft.com][coc_contact] with any additional questions or comments.
+
+<!-- LINKS -->
+[cla]: https://cla.microsoft.com
+[code_of_conduct]: https://opensource.microsoft.com/codeofconduct/
+[coc_faq]: https://opensource.microsoft.com/codeofconduct/faq/
+[coc_contact]: mailto:opencode@microsoft.com
+[azure_subscription]: https://azure.microsoft.com/free/
+[cognitive_resource_cli]: https://docs.microsoft.com/azure/cognitive-services/cognitive-services-apis-create-account-cli
+[jdk_link]: https://docs.microsoft.com/java/azure/jdk/?view=azure-java-stable
+
+<!--
+[cancer_profiling_client_class]: https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/healthinsights/azure-health-insights-cancerprofiling/src/main/java/com/azure/health/cancerprofiling/CancerProfilingClient.java
+-->
+[azure_cli]: https://docs.microsoft.com/cli/azure
+[azure_portal]: https://portal.azure.com
+[cancer_profiling_docs]: https://review.learn.microsoft.com/azure/cognitive-services/health-decision-support/oncophenotype/overview?branch=main
```

Comment:
nit, you may want to add an impression png

e.g. last line of https://raw.githubusercontent.com/Azure/azure-sdk-for-java/main/sdk/deviceupdate/azure-iot-deviceupdate/README.md

Reply:
![image](https://user-images.githubusercontent.com/87355844/228468910-e9d25024-d631-4676-87ad-20efeab07131.png)
This impression is like this, is it expected?

Reply:
I don't see any problem?

https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-java%2Fsdk%healthinsights%2Fazure-health-insights-cancerprofiling%2FREADME.png